code/datums/lp_mission_holotapes.dm

```diff
@@ -0,0 +1,299 @@
+/obj/item/disk/holodisk/lp/combat/syndicate1
+	name = "Combat Mission Eliminate Force"
+	desc = "A holodisk containing a combat mission for the LP."
+	preset_image_type = /datum/preset_holoimage/commissioner
+	preset_record_text = {"
+	NAME Commissioner Gorre Donn
+	SAY This mission, should you choose to accept it, revolves around a combat objective. The LP will have to get their hands dirty for this one.
+	DELAY 25
+	SAY The LP is to respond to a hostile threat from a Syndicate vessel. In such action they are to kill or otherwise subdue the primary fighting force.
+	DELAY 25
```

Comment:
tab these one more please

Reply:
for all of em

Reply:
![image](https://user-images.githubusercontent.com/55075747/178066071-3c1590e0-2401-4a38-9ff6-dd71a4b954f0.png)
This is how they're formatted by mark and anturk in the holocall file.
![image](https://user-images.githubusercontent.com/55075747/178066170-fccdf97d-61e6-43f9-8ace-a05038eb4d0d.png)

Reply:
sure i guess